src/metaopt/cli.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+"""
+mopt:
+  MetaOpt cli script for asynchronous distributed optimization
+
+ ERROR CODES
+-------------
+0  : Success
+-1 : Provide a name for the experiment (cli argument or moptconfig file)
+
+"""
+from __future__ import absolute_import
+
+import datetime
+import getpass
+import logging
+import sys
+
+from metaopt import resolve_config as resolvconf
```

Comment:
Why do you rename it? Would we agree to follow [pep8 naming's convention](https://www.python.org/dev/peps/pep-0008/#function-names)?

Reply:
Yes, we will follow pep8 naming convention. However, this is not against pep8. I noticed that in many projects (even in Python's stl) ppl really avoid underscored names if it is clear enough, e.g. `configparser`, `argparse`, `importlib`, `modulefinder`, `pkgutil`, etc.

Eitherways, `resolvconf` is a commonly recognisable abbreviation due to being used in linux jargon (https://duckduckgo.com/?q=resolvconf&ia=web). I renamed it to be shorter when I use it 😛 .

Reply:
I actually prefer that modules avoid having underscores in their names, but this is due to my personal linting preferences. 😝

Reply:
Yes pep8 is flexible 

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.

However I don't like relying on outside's context for variable name clarity. I must say I do not understand the argument for short variable names nowadays. Every text editor, even vim, has auto-completion add-ons so we virtually never type a full variable name more than once. But god knows how many times we read them. :stuck_out_tongue: